utils/interfaces/_test_agent.py

```diff
@@ -56,6 +56,17 @@ def get_telemetry_for_runtime(self, runtime_id):
 
         return telemetry_msgs
 
+    def get_crashlog_for_runtime(self, runtime_id):
+        logger.debug(f"Try to find a crashlog related to runtime-id {runtime_id}")
+        assert runtime_id is not None, "Runtime ID not found"
+        telemetry_msgs = []
+        for data_received in self._data_telemetry_list:
+            print(data_received)
+            if data_received["request_type"] == "logs" and data_received["runtime_id"] == runtime_id:
+                telemetry_msgs.append(data_received)
+
+        return telemetry_msgs
```

Comment:
```suggestion
        return [l for l in self.get_telemetry_logs() if l["runtime_id"] == runtime_id]
```

I think this is the same?


I didn't realize we had duplicated some helpers, the `get_crash_reports` helper below also checks if we have crash tags, otherwise we might just be grabbing some random telemetry logs that aren't crash reports.


Not sure exactly how much this matters, but .... 🤷🏻